Cordova.podspec

```diff
@@ -0,0 +1,32 @@
+Pod::Spec.new do |s|
+  s.name         = "Cordova"
+  s.version      = "5.1.0"
+  s.summary      = "Cordova for iOS"
+  s.homepage     = "https://github.com/apache/cordova-ios"
+  s.license      = { :type => "Apache 2.0", :file => "LICENSE" }
+  s.author       = { "Bharath Hariharan" => "bhariharan@salesforce.com" }
```

Comment:
Is this correct? Does this refer to who created the podspec or who "wrote"/owns the software?

Reply:
This refers to the original author.

Reply:
It should be "Apache Software Foundation"